composer.json

```diff
@@ -90,7 +90,8 @@
         "ci": [
             "@ci:static",
             "@ci:dynamic"
-        ]
+        ],
+        "phive:update:phpunit": "echo y | \"./tools/phive.phar\" update phpunit"
```

Comment:
I propose we use the phive option `--no-progress` here as the progress bar does not really work when the output is logged.

And maybe we can explicitly trust Sebastian's GPG key D8406D0D82947747293778314AA394086372C20A by using the `--trust-gpg-keys` option instead of consenting to the first question phive asks.

Reply:
I could not find the documentation for PHIVE so did not know about these, but have now added the command line options.

Reply:
I could not find any documentation for PHIVE so did not know about these options.  However, the `--trust-gpg-keys` option appears to be unknown - I don't know if it requires PHIVE version 0.14 - I selected version 0.13.5 because that is compatible with PHP 7.1 (although we are only actually running it during the build for PHP >= 7.3).

Reply:
Executing ```phive``` without any options should display its built-in help.

Reply:
Then I'm all for using the latest version of PHIVE so we can use the current features and bug fixes.